lms/djangoapps/courseware/access.py

```diff
@@ -314,6 +314,10 @@ def _can_enroll_courselike(user, courselike):
     # which actually points to a CourseKey. Sigh.
     course_key = courselike.id
 
+    if settings.FEATURES.get('RESTRICT_ENROLL_NO_ATSIGN_USERNAMES') and '@' in user.username:
+        log.warning("ENROLLMENT DENIED in %s because @ in username %s" % (course_key, user.username))
+        return ACCESS_DENIED
+
```

Comment:
Yeah, this is a change that is unrelated to this PR. It is most likely an artifact of the branch that this was based off of.